orion/sites/admin/src/index.js

```diff
@@ -51,6 +54,8 @@ class App extends React.Component {
           <Route path="/program/:programId/edit" component={ProgramEditMatch}/>
           <Route path="/programs/add" component={ProgramEdit}/>
           <Route path="/programs" component={Programs}/>
+          <Route path="/announce/:announceId/edit" component={AnnounceEditMatch}/>
+          <Route path="/announce/add" component={AnnounceEdit}/>
```

Comment:
Ah another thought, below we have `"/announcements"` as a path. Since these all do with announcements, let's make them consistent. For the other paths, can you make them:
```
"/announcements/:announceId/edit"
"/announcements/add"
```

Thanks!